result.js

```diff
@@ -87,11 +105,18 @@ const getAllSongsInfo = response => {
 
 const searchASong = response => {
   return new Promise(resolve => {
-    get("https://api.spotify.com/v1/search", {
-      Authorization: response.token_type + ' ' + response.access_token
-    }, buildSearchQuery(response.song), responseFromSearch => {
-      resolve(responseFromSearch.tracks.items[0]);
-    });
+      var waittime = 100; // milliseconds between calls
```

Comment:
Thanks A LOT for adding the 100 ms delay for each call as well as making it call sequentially instead of All in one. This fix should solve this problem, https://github.com/poanchen/google-play-music-playlist-exporter/issues/5 (facepalm) for not able to use timeout method in JS.